modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CheckpointCommand.java

```diff
@@ -17,25 +17,33 @@
 package org.apache.ignite.internal.commandline;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.UUID;
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 import org.apache.ignite.internal.client.GridClient;
 import org.apache.ignite.internal.client.GridClientCompute;
 import org.apache.ignite.internal.client.GridClientConfiguration;
 import org.apache.ignite.internal.client.GridClientDisconnectedException;
+import org.apache.ignite.internal.client.GridClientException;
 import org.apache.ignite.internal.client.GridClientNode;
 import org.apache.ignite.internal.util.typedef.F;
+import org.apache.ignite.internal.util.typedef.internal.SB;
 import org.apache.ignite.internal.visor.VisorTaskArgument;
 import org.apache.ignite.internal.visor.checkpoint.VisorCheckpointTask;
 import org.apache.ignite.internal.visor.checkpoint.VisorCheckpointTaskResult;
+import org.jetbrains.annotations.Nullable;
 
 import static org.apache.ignite.internal.commandline.Command.usage;
 import static org.apache.ignite.internal.commandline.CommandList.CHECKPOINT;
 
 /**
- * Command to run checkpoint on cluster
+ * Command to run checkpoint on the cluster.
  */
 public class CheckpointCommand extends AbstractCommand<Void> {
+    /** */
+    private static final String NODE_ID_ARG_NAME = "--node-id";
+
     /** {@inheritDoc} */
     @Override public void printUsage(Logger log) {
         usage(log, "Start checkpointing process:", CHECKPOINT);
```

Comment:
Please update the "usage" message, it should reflect the presence of new parameter

Reply:
Updated usage